apps/www/src/registry/default/example/playground-demo.tsx

```diff
@@ -216,6 +218,10 @@ export const usePlaygroundPlugins = ({
             ...selectOnBackspacePlugin,
             enabled: !!enabled.selectOnBackspace,
           }),
+          createRemoveOnDeleteForwardPlugin({
+            ...removeOnDeleteForwardPlugin,
```

Comment:
No default

Reply:
I was thinking we could apply it to only paragraphs by default in the playground, just in case we have weird behaviour in nested structures.

From my earlier comment:
> In the playground, can you use the query option to only apply the plugin when inside paragraph elements?

Reply:
for playground, I am enabling it for paragraphs, for now. I hope thats fine?

Reply:
@12joan Default behavior should already include `ELEMENT_DEFAULT`, so we would not need any option in the playground. What do you think?

Reply:
removed the parameter, its working with default values now.

Reply:
You'll need to include the default query (only allowing `ELEMENT_DEFAULT`) in an `options` object of `createDeletePlugin`. See other plugins for inspiration.